code/src/components/ThoughtForm.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+const ThoughtForm = ({ onFormSubmit, newThought, setNewThought, letterCounter }) => {
+
+
+    return (
+        <form className="form" onSubmit={onFormSubmit}>
+            <label htmlFor='newThought'className='input-label'>What is happiness for you?</label>
```

Comment:
Happiness is EVERYTHING! Life.. Family.. Good Food.. yeah it's very important 😄

Reply:
❤️